From 7be8cae20d50d1d991bf5a0ad0155e0f784b0b60 Mon Sep 17 00:00:00 2001
From: James Hilliard <james.hilliard1@gmail.com>
Date: Tue, 1 Oct 2024 10:24:56 -0600
Subject: [PATCH] Add support for weston 16 protocols

Weston 15 was released recently and the weston development branch is
now using libweston-16-protocols.

Signed-off-by: James Hilliard <james.hilliard1@gmail.com>

Upstream: https://github.com/Igalia/cog/pull/789

Signed-off-by: Bernd Kuhls <bernd@kuhls.net>
---
 platform/wayland/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/platform/wayland/meson.build b/platform/wayland/meson.build
index dad47a6..3ecf9d8 100644
--- a/platform/wayland/meson.build
+++ b/platform/wayland/meson.build
@@ -49,6 +49,7 @@ wayland_protocols_path = wayland_protocols_dep.get_variable(pkgconfig: 'pkgdatad
 
 if wayland_platform_weston_protocols.length() > 0
     foreach weston_dep_name : [
+            'libweston-16-protocols',
             'libweston-15-protocols',
             'libweston-14-protocols',
             'libweston-13-protocols',
-- 
2.47.3

